packages/relay/src/lib/clients/mirrorNodeClient.ts

```diff
@@ -911,4 +911,20 @@ export class MirrorNodeClient {
     public getMirrorNodeWeb3Instance(){
         return this.web3Client;
     }
+
+    /**
+     * This method is intended to be used in cases when the default axios-retry settings do not provide
+     * enough time for the expected data to be propagated to the Mirror node.
+     * It provides a way to have an extended retry logic only in specific places
+     */
+    public async repeatedRequest(methodName: string, args: any[], repeatCount: number, requestId?: string) {
+        let result;
+        for (let i = 0; i < repeatCount; i++) {
+            result = await this[methodName](...args, requestId);
+            if (result) {
+                break;
+            }
+        }
+        return result;
+    }
 }
```

Comment:
I like this method, I would like if we could disable the axios default retries. 
why? some logic flows are indeed expecting a `404` code as part of their conditions, and in those cases the axios default retry is adding a 3s latency unneeded before returning the expected and correct `404` code.
Also I think that the retries should be intended on certain cases and codes in specific logic flows and not for all methods as is currently with the axios interceptor implementation.
So I think this method takes in that direcction and once we use it with intention on the needed places, we can just set the amount of retries to 0 and effectively disabled it.

Reply:
Another thing, it might not be needed right now, but I think the method should have a try..catch and identify `404`.
Right now, only some methods might `throw` an exception instead of returning `null` when receiving a `404` from the mirrorNode, although I believe that the `mirrorNodeClient` should always return the `404` and let the consuming method to handle the `404` as it sees fit.